conf/routes

```diff
@@ -53,7 +53,7 @@ GET            /capi/*path                                 controllers.CAPIServi
 
 # Content
 GET            /api/v1/content/:id                         controllers.ContentApi.contentById(id: String)
-
+GET            /api/v1/content/editor/:id                  controllers.ContentApi.contentByEditorId(id: String)
```

Comment:
It might be good to keep the same naming convention from `workflow`: `/api/atom/:editorId`.

I'm not sure where this v1 api is used.